internal/server/server.go

```diff
@@ -65,6 +63,10 @@ func (s *Server) Run() error {
 	s.log.Println("Initializing config providers")
 	_ = git.NewGitConfigProvider(cache, cacheExpirationTime, cacheExpirationTime)
 
+	s.log.Println("Initializing async jobs")
+	s.taskManager = tasks.Init(s.log, s.store)
```

Comment:
minor: since its local no need to add to server 
```suggestion
taskManager := tasks.Init(s.log, s.store)
```

Reply:
Done